libs/application/templates/funding-government-projects/src/lib/dataSchema.ts

```diff
@@ -1,7 +1,24 @@
 import * as z from 'zod'
 
+const emailRegex = /^[\w!#$%&'*+/=?`{|}~^-]+(?:\.[\w!#$%&'*+/=?`{|}~^-]+)*@(?:[A-Z0-9-]+\.)+[A-Z]{2,6}$/i
+const isValidEmail = (value: string) => emailRegex.test(value)
+import { error } from './messages'
+
 export const FundingGovernmentProjectsSchema = z.object({
-  field: z.string(),
+  organizationOrInstitutionName: z
+    .string()
+    .nonempty(error.general.nameError.defaultMessage),
+  contacts: z.array(
+    z.object({
+      name: z.string().nonempty(error.general.nameError.defaultMessage),
+      phoneNumber: z.string().refine((v) => v && v.length >= 7, {
+        params: error.general.invalidPhoneNumber,
+      }),
+      email: z.string().refine((v) => isValidEmail(v), {
```

Comment:
This can be replaced by setting the text field variant as email

Reply:
Validation seems doesnt seem to work without some validation in data schema.